Dockerfile-fpm

```diff
@@ -8,6 +8,8 @@ ENV PHP_FPM_PM_MIN_SPARE_SERVERS=1
 ENV PHP_FPM_PM_MAX_SPARE_SERVERS=3
 ENV PHP_FPM_PM_PROCESS_IDLE_TIMEOUT=10
 ENV PHP_FPM_PM_MAX_REQUESTS=0
+ENV PHP_FPM_LISTEN_BACKLOG=511
+ENV PHP_FPM_LISTEN_URI=/var/run/php-fpm.sock
```

Comment:
I think this can be better if we do:

```
ENV PHP_FPM_LISTEN_URI=/var/run/php-fpm.sock
ENV FCGI_CONNECT=$PHP_FPM_LISTEN_URI
ENV PHP_FPM_PM=dynamic	ENV PHP_FPM_PM=dynamic
ENV PHP_FPM_PM_MAX_CHILDREN=5	ENV PHP_FPM_PM_MAX_CHILDREN=5
ENV PHP_FPM_PM_START_SERVERS=2	ENV PHP_FPM_PM_START_SERVERS=2
ENV PHP_FPM_PM_MIN_SPARE_SERVERS=1	ENV PHP_FPM_PM_MIN_SPARE_SERVERS=1
ENV PHP_FPM_PM_MAX_SPARE_SERVERS=3	ENV PHP_FPM_PM_MAX_SPARE_SERVERS=3
ENV PHP_FPM_PM_PROCESS_IDLE_TIMEOUT=10	ENV PHP_FPM_PM_PROCESS_IDLE_TIMEOUT=10
ENV PHP_FPM_PM_MAX_REQUESTS=0	ENV PHP_FPM_PM_MAX_REQUESTS=0
ENV PHP_FPM_LISTEN_BACKLOG=511
```